src/styles/main.scss

```diff
@@ -1,3 +1,25 @@
+@font-face {
+  font-family: "Roboto", sans-serif;
+  src: url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
```

Comment:
download ttf file and set here or instead font-face set right link to html file
  ```
    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
    rel="stylesheet">
  ```
check all styles for font
fonts still missmatch with design and tests failed